client/flutter/lib/pages/latest_numbers.dart

```diff
@@ -0,0 +1,144 @@
+import 'package:WHOFlutter/api/who_service.dart';
+import 'package:WHOFlutter/components/page_button.dart';
+import 'package:WHOFlutter/components/page_scaffold/page_scaffold.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+import 'package:WHOFlutter/generated/l10n.dart';
+import 'package:intl/intl.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+const number = TextStyle(
+  color: Color(0xFFAF2B2B),
+  fontSize: 36,
+);
+const name = TextStyle(
+  color: Colors.black,
+  fontSize: 16,
+  fontWeight: FontWeight.w700,
+);
+const header =
+    TextStyle(color: Colors.black, fontSize: 24, fontWeight: FontWeight.w800);
+
+class LatestNumbers extends StatelessWidget {
+  _launchStatsDashboard(BuildContext context) async {
+    var url = S.of(context).homePagePageButtonLatestNumbersUrl;
+    if (await canLaunch(url)) {
+      await launch(url);
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final S localized = S.of(context);
+
+    return PageScaffold(context,
+        title: "Latest Numbers",
+        showShareBottomBar: false,
+        body: [
+          FutureBuilder(
+              future: WhoService.getCaseStats(),
+              builder: (context, snapshot) {
+                final ts = snapshot.hasData ? new DateTime.fromMillisecondsSinceEpoch(snapshot.data['globalStats']['lastUpdated']) : DateTime.now();
```

Comment:
out of scope on this PR, but probably a good idea longer term to do some validation that the data exists in this structure across all of the 2nd level keys (e.g. ensuring `snapshot.data['globalStats']` is not null before trying to get a key on it)

Reply:
Will fix now.